test/expected.txt

```diff
@@ -184,8 +184,9 @@
 || 0 persons found!
 || ===================================================
 || Enter command: || [Command entered:  find betsy]
+|| 	1. Betsy Choo  Phone Number: 222222  Email: benchoo@nus.edu.sg
```

Comment:
Good job in changing the tests. 
When you modify the behavior, remember to update documentation well.